src/api/labels.test.js

```diff
@@ -0,0 +1,93 @@
+import { checkLabel, saveLabel, saveName } from './labels'
+const KEY = 'labels'
+
+beforeEach(() => {
+  localStorage.clear()
+})
+
+function setupStorage() {
+  localStorage.__STORE__[KEY] = JSON.stringify({
+    '0x36940f34a2ec6afe46b7db53e6611470cd76c4f5999209a04a670682e2c33f75':
+      'blahblah'
+  })
+}
+
+function getLabelsFromStorage() {
+  return JSON.parse(localStorage.__STORE__[KEY])
+}
+
+describe('checkLabel', () => {
+  test('should return label in localStorage', () => {
+    setupStorage()
+    const hash =
+      '0x36940f34a2ec6afe46b7db53e6611470cd76c4f5999209a04a670682e2c33f75'
+    expect(
+      checkLabel(
+        '0x36940f34a2ec6afe46b7db53e6611470cd76c4f5999209a04a670682e2c33f75'
+      )
+    ).toBe('blahblah')
+  })
+
+  test('should return undefined if label is not in localStorage', () => {
+    setupStorage()
+    const hash =
+      '0x36940f34a2ec6afe46b7db53e6611470cd76c4f5999209a04a670682e2c33f75'
+    expect(
+      checkLabel(
+        '0x36940f34a2ec6afe46b7db53e6611470cd76c4f5999209a04a670682e2c33f74'
```

Comment:
I can only figure out that this hash is different to the one at line 34, by looking at the last digit. Please turn it into constant/variable so easier to know they are different